llvm/lib/CodeGen/SelectionDAG/SelectionDAG.cpp

```diff
@@ -4212,7 +4212,6 @@ KnownBits SelectionDAG::computeKnownBits(SDValue Op, const APInt &DemandedElts,
     break;
   }
 
-  assert(!Known.hasConflict() && "Bits known to be one AND zero?");
```

Comment:
Maybe just focus on the middle-end initially and then look at removing the DAG/GlobalISel assertions later?

Reply:
I feel like it's safer to get rid of it everyone at once.